src/components/content/order/common/RegionSelection.tsx

```diff
@@ -39,15 +40,22 @@ export const RegionSelection = ({
                 <Space wrap>
                     <Select
                         className={serviceOrderStyles.selectBoxClass}
-                        defaultValue={selectRegion}
-                        value={selectRegion}
+                        defaultValue={selectRegion.name + '(' + selectRegion.site + ')'}
+                        value={selectRegion.name + '(' + selectRegion.site + ')'}
                         onChange={(newRegion) => {
                             if (onChangeRegion) {
-                                onChangeRegion(newRegion);
+                                const regionName: string = newRegion.split('(')[0];
+                                const siteName: string = newRegion.split('(')[1].split(')')[0];
+                                const newRegionObject: Region = {
+                                    name: regionName.trim(),
+                                    site: siteName.trim(),
+                                    area: '',
```

Comment:
its better if we can send the area here instead of empty.

Reply:
Updated the area value with 'selectArea'